coreneuron/nrnconf.h

```diff
@@ -31,7 +31,10 @@ using Symbol = char;
 #define VEC_AREA(i) (_nt->_actual_area[(i)])
 #define VECTORIZE   1
 
+// extern variables require acc declare
 extern double celsius;
+#pragma acc declare create(celsius)
+
 extern double pi;
```

Comment:
Should we do the same for `pi`?

Reply:
At the moment `pi` is only used in `lfp.cpp`, so there is no need for it to be allocated here on the GPU, however if it's used at some point in mod files we should add more logic to update in in GPU in mod2c/NMODL

Reply:
Yes. I did the change for `pi` as well and then revered it because `pi` is typically not used (?). But I can change it.